perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -288,6 +296,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
   .toolbarButton {
     min-width: auto;
+    float: right;
```

Comment:
Please make sure to enable RTL when adding float

Reply:
RP added : https://github.com/Meeds-io/perk-store/pull/47 !! @ahamdi